runtime/index.d.ts

```diff
@@ -1,24 +1,40 @@
-declare module '@sapper/app' {
+import { Readable, Writable } from "svelte/store";
+import { PageContext } from "@sapper/app/types";
+
+declare module "@sapper/app";
```

Comment:
there were just removed in https://github.com/sveltejs/sapper/pull/1598. you might need to rebase against `master`